app/controllers/kittens_controller.rb

```diff
@@ -7,4 +7,30 @@ def index
 
     render layout: true
   end
+
+  def new
+    # TODO
+    # @kitten = Kitten.new
```

Comment:
No, plugins are effectively enhanced Rails engines and thus migrations of the core also run all registered plugin migrations. This is done internally by the OP/redmine plugin bridge by adding a plugin's `db/migrate` path to the Rails migrator.